Library/Homebrew/formulary.rb

```diff
@@ -121,22 +123,23 @@ def initialize(name, path)
     #
     # `alias_path` can be overridden here in case an alias was used to refer to
     # a formula that was loaded in another way.
-    def get_formula(spec, alias_path: nil)
-      klass.new(name, path, spec, alias_path: alias_path || self.alias_path)
+    def get_formula(spec, alias_path: nil, force_bottle: false, flags: [])
+      klass(flags: flags)
+        .new(name, path, spec, alias_path: alias_path || self.alias_path, force_bottle: force_bottle)
     end
```

Comment:
```suggestion
    def get_formula(spec, alias_path: self.alias_path, force_bottle: false, flags: [])
      klass(flags: flags).new(name, path, spec, alias_path: alias_path, force_bottle: force_bottle)
    end
```

Reply:
Seems like this doesn't quite do what you would expect.

Reply:
Some other methods explicitly pass `alias_path: nil`, overriding it if it is in the signature.

Reply:
Doh. 
```suggestion
    def get_formula(spec, alias_path: nil, force_bottle: false, flags: [])
      alias_path ||= self.alias_path
      klass(flags: flags).new(name, path, spec, alias_path: alias_path, force_bottle: force_bottle)
    end
```